frontend/javascripts/oxalis/view/layouting/tracing_layout_view.tsx

```diff
@@ -53,6 +53,7 @@ const { Sider } = Layout;
 type OwnProps = {
   initialMaybeCompoundType: APICompoundType | null;
   initialCommandType: TraceOrViewCommand;
+  UITheme: Theme;
```

Comment:
I'd change it to `uiTheme`. or is there a specific reason to not use camel case?